installer/packer/scripts/kov/start_kov.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/bash
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -euf -o pipefail
+
+KOVD_EXPOSED_PORT=""
+KOVD_KEY_LOCATION=""
+KOVD_CERT_LOCATION=""
+
+/usr/bin/kovd --tls-port=${KOVD_EXPOSED_PORT} --tls-host=0.0.0.0 --tls-certificate ${KOVD_CERT_LOCATION} --tls-certificate-key ${KOVD_KEY_LOCATION}
```

Comment:
You do not need this file.
if you do insist on keeping it, it needs to have an exec in front of this /usr/bin/kovd